src/chromo/common.py

```diff
@@ -602,18 +531,10 @@ def __init__(self, seed):
         assert hasattr(self, "_frame")
         self._lib = importlib.import_module(f"chromo.models.{self._library_name}")
 
-        if seed is None:
-            self._seed = randint(1, 10000000)
-        elif isinstance(seed, int):
-            self._seed = seed
-        else:
-            raise ValueError(f"Invalid seed {seed}")
-
-        if hasattr(self._lib, "init_rmmard"):
-            self._lib.init_rmmard(self._seed)
-
-        # TODO use rmmard for this, too, instead of numpy PRNG
-        self._composite_target_rng = np.random.default_rng(self._seed)
+        self._seed = seed
+        self._rng = np.random.default_rng(seed)
+        if hasattr(self._lib, "npy"):
```

Comment:
Is there any model for which seed should be set in different place, i.e. it doesn't have "npy" common block?

Reply:
Pythia8 is the only one. For Pythia8 we have to do it differently, everyone else uses the code in rangen.fpp which uses the npy common block.

Reply:
Pythia8 is currently not using the numpy PRNG. I have to see how this can be achieved for Pythia8, that's for another PR.